tests/data/test_data_preprocessing_utils.py

```diff
@@ -384,3 +389,52 @@ def test_process_dataargs_pretokenized(data_args):
     assert set(["input_ids", "labels"]).issubset(set(train_set.column_names))
     if eval_set:
         assert set(["input_ids", "labels"]).issubset(set(eval_set.column_names))
+
+
+@pytest.mark.parametrize(
+    "datafile, column_names, datasetconfigname",
+    [
+        (
+            TWITTER_COMPLAINTS_DATA_INPUT_OUTPUT_JSON,
+            set(["ID", "Label", "input", "output"]),
+            "text_dataset_input_output_masking",
+        ),
+        (
+            TWITTER_COMPLAINTS_TOKENIZED_JSON,
+            set(
+                [
+                    "Tweet text",
+                    "ID",
+                    "Label",
+                    "text_label",
+                    "output",
+                    "input_ids",
+                    "labels",
+                ]
+            ),
+            "pretokenized_dataset",
+        ),
+        (
+            TWITTER_COMPLAINTS_DATA_JSON,
+            set(["Tweet text", "ID", "Label", "text_label", "output"]),
+            "apply_custom_data_template",
+        ),
+    ],
+)
+def test_process_dataset_configs(datafile, column_names, datasetconfigname):
+    """Test process_dataset_configs for expected output."""
+    dataloaderconfig = DataLoaderConfig(type="default")
```

Comment:
I think `DataLoaderConfig(type="default")` would be same as `DataLoaderConfig()`.

Reply:
that's true, thanks